Source/WebCore/platform/graphics/cocoa/SystemFontDatabaseCoreText.cpp

```diff
@@ -51,12 +51,6 @@ SystemFontDatabaseCoreText::SystemFontDatabaseCoreText() = default;
 
 RetainPtr<CTFontRef> SystemFontDatabaseCoreText::createSystemUIFont(const CascadeListParameters& parameters, CFStringRef locale)
 {
-    // Work around a quirk of the platform API.
```

Comment:
I believe that the fix for this workaround only shipped in macOS 15.4/iOS 18.4. Since we build WebKit for the two prior releases (which still have this bug) we need to keep this.

Please use a compile guard here:

```
#if (PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED < 150400)
...
#endif
```